clients/cli/src/system.rs

```diff
@@ -1,84 +1,66 @@
 //! System information and performance measurements
 
-use rayon::prelude::*;
-use std::hint::black_box;
+use cfg_if::cfg_if;
 use std::process;
 use std::thread::available_parallelism;
-use std::time::Instant;
-use sysinfo::System;
+use sysinfo::{CpuRefreshKind, RefreshKind, System};
 
 /// Get the number of logical cores available on the machine.
 pub fn num_cores() -> usize {
     available_parallelism().map(|n| n.get()).unwrap_or(1) // Fallback to 1 if detection fails
 }
 
-/// Total memory in GB of the machine.
```

Comment:
I moved these down in the file, so that CPU-related functions and memory-related functions are grouped together